src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/ValidationHelper.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2024 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.bigquery.stepsdesign;
```

Comment:
Also Validation helper usually go in utils or common folder, are we adding Validation helpers in specific modules for other plugins as well ?

Reply:
No, we don't add in utils. We simply add them in module folder.